mountpoint-s3/tests/fuse_tests/prefetch_test.rs

```diff
@@ -85,16 +84,16 @@ where
 #[test_case(512 * 1024, 1; "first request size greater than seek position reading 1 byte")]
 #[test_case(256 * 1024, 200 * 1024; "default first request size reading till seek")]
 fn prefetch_test_etag_mock(request_size: usize, read_size: usize) {
-    // prefetch_test_etag(
-    //     crate::fuse_tests::mock_session::new,
-    //     "prefetch_etag_test",
-    //     request_size,
-    //     read_size,
-    // );
+    prefetch_test_etag(
+        crate::fuse_tests::mock_session::new,
+        "prefetch_etag_test_mock",
+        request_size,
+        read_size,
+    );
 
     prefetch_test_etag(
```

Comment:
I think the pattern we are using elsewhere is to have 2 functions, see for example write_tests.rs. 

In this case, they would be: 
* `fn prefetch_test_etag_mock` 
* `fn prefetch_test_etag_s3` with `#[cfg(feature = "s3_tests")]`

That means there can be some duplication in the `test_case` attributes, but in general they do not need to be the same.